### PR TITLE
Update pacman-fix-permissions  - fixes #1

### LINKED ISSUE #1: BUG: file not found
```
(100/912) dislocker 0.7.1-2
=> dislocker package is missing, downloading
Traceback (most recent call last):
  File "/usr/bin/pacman-fix-permissions", line 79, in <module>
    main()
  File "/usr/bin/pacman-fix-permissions", line 57, in main
    old_mode = int(lstat(p).st_mode & 0o7777)
FileNotFoundError: [Errno 2] No such file or directory: '/usr/share/dislocker'
```

### PR DESCRIPTION
**Pull request by Prathik-Jain**
Added an try-except block In order to overcome files which the user has moved intentionally / unintentionally like  `node_modules`

Thanks for the script ! :+1:  :100:

### CODE REVIEW

**Comment by droserasprout** [comment]
Thank you, @Prathik-Jain!

File: pacman-fix-permissions [hunk 1]
```diff
@@ -61,7 +61,7 @@ def main():
                             paths[p] = {'old_mode': old_mode,
                                         'new_mode': new_mode}
                     except FileNotFoundError as e:
-                        print("File not found: {}".format(e))
+                        logger.error('File not found: {}'.format(p))
```
Comment (Prathik-Jain) [inline pacman-fix-permissions]: `p`?

Comment (droserasprout) [reply pacman-fix-permissions]: It's current path. For ex.
```
(162/1382) filesystem 2018.1-2
File not found: /var/spool/mail
```

Comment (Prathik-Jain) [reply pacman-fix-permissions]: :+1:

**Pull request merged**